.github/workflows/configure.cmd

```diff
@@ -25,6 +25,7 @@ set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\StatisticsApplicat
 set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\SwimmingDEMApplication;
 set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\ShapeOptimizationApplication;
 set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\ConstitutiveLawsApplication;
+REM set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%KRATOS_APP_DIR%\RANSApplication;
```

Comment:
can we revert this back on? Afterwards I can approve. Thanks a lot @roigcarlo. I'll change the test_utilities.h in a sepearte PR.

Reply:
Saldy no, we have to merge with this disabled, once its merge I will activate it in another PR. Its causing problems with the cache dor some reason...